frontend-ssr/pages/demo/html/fixed-size-image.tsx

```diff
@@ -29,6 +29,7 @@ export default function FixedSizeImage() {
               display: 'flex',
```

Comment:
I think this can be slightly refactored to use classes:`d-flex`, `align-items-center` etc. 

I know I wrote the code that way,  but the takeaway here is if you see something that can be improved, don't hesitate to improve it.

Reply:
Ok, got it.